nautobot/extras/models/models.py

```diff
@@ -166,11 +166,14 @@ class CustomLink(BaseModel, ChangeLoggedModel):
         limit_choices_to=FeatureQuery("custom_links"),
     )
     name = models.CharField(max_length=100, unique=True)
-    text = models.CharField(max_length=500, help_text="Jinja2 template code for link text")
+    text = models.CharField(
+        max_length=500,
+        help_text="Jinja2 template code for link text. Reference the object as '{{ obj }}'. Links which render as empty text will not be displayed.",
```

Comment:
IMO this would be better:

```suggestion
        help_text="Jinja2 template code for link text. Reference the object as <code>{{ obj }}</code>. Links which render as empty text will not be displayed.",
```
![Cursor_and_Add_a_new_custom_link_-_Nautobot](https://user-images.githubusercontent.com/138052/111042537-ae07c080-83f2-11eb-9a70-e78317c7f2c0.png)